docker/Dockerfile

```diff
@@ -214,6 +214,13 @@ RUN wget --progress=dot:giga \
         && rm v_linux.zip
 ENV PATH="/opt/v/:${PATH}"
 
+# https://ziglang.org/download
+ARG ZIG=zig-linux-x86_64-0.8.0
```

Comment:
It's not required to use the stable versions, it's okay to use the development versions as soon as we can track them. But it's up to you.

Reply:
I've tried nightly version, but that wasn't making any difference. Besides, I don't know long they are keeping nightly releases for. Might be better for now to stick to stable release.